core/lib/dal/src/blocks_dal.rs

```diff
@@ -804,33 +804,12 @@ impl BlocksDal<'_, '_> {
         if update_result.rows_affected() == 0 {
             tracing::debug!("L1 batch #{number}: tree data wasn't updated as it's already present");
 
-            // Batch was already processed. Verify that existing hash matches
-            let matched: i64 = sqlx::query!(
```

Comment:
Is there a particular reason the tree data was checked like this? I've simplified it just to use `get_l1_batch_tree_data()`.

Reply:
Probably historical reasons(tm).